src/input/Input.tsx

```diff
@@ -122,6 +122,7 @@ export const InputField = forwardRef<
             type={type}
             checked={checked}
             disabled={disabled}
+            aria-describedby={description ? id + "-desc" : undefined}
```

Comment:
`id` could be undefined, so I'd suggest generating one. react-aria has a `useId` hook: https://react-spectrum.adobe.com/react-aria/useId.html

Reply:
Ah, useful - thanks.